shared/components/page-header/__stories__/index.jsx

```diff
@@ -0,0 +1,60 @@
+import React from 'react';
+import { storiesOf } from '@storybook/react';
+import { withInfo } from '@storybook/addon-info';
+
+import PageHeader from '../PageHeader';
+
+const count = [].length;
+
+storiesOf('PageHeader', module)
+  .addDecorator(withInfo)
+  .add(
+    'Time Entries',
+    () => (
+      <PageHeader
+        title="Timesheets"
+        subtitle={`${count} ${count === 1 ? 'Entry' : 'Entries'}`}
```

Comment:
Having count statically at `0` it doesn't make sense to add this if-statement here.